### PR TITLE
fixing missing None and edge-cases line empty lines

### PR DESCRIPTION
**Pull request by lamastex**
Found these bugs when doing parsing across multiple days.

### CODE REVIEW

File: src/main/scala/com/gravity/goose/spark/GooseFetcher.scala [hunk 1]
```diff
@@ -0,0 +1,123 @@
+package com.gravity.goose.spark
```
Comment (aamend) [inline src/main/scala/com/gravity/goose/spark/GooseFetcher.scala:1]: Sorry for breaking this interface in previous release. I stay convinced that goose scraper must be brought as external deps to the project that we can explicitly ignore when not required. We'll keep it for now but may consider moving that elsewhere

File: pom.xml [hunk 2]
```diff
@@ -1,270 +1,320 @@
...
+  <groupId>com.aamend.spark</groupId>
+  <artifactId>spark-gdelt</artifactId>
+  <version>2.2-SNAPSHOT</version>
+  <url>https://github.com/aamend/spark-gdelt</url>
+  <description>Working with GDELT from Spark
+  environment</description>
+  <inceptionYear>2015</inceptionYear>
+  <developers>
+    <developer>
+      <id>aamend</id>
```
Comment (aamend) [inline pom.xml:17]: @lamastex please put your name in alongside your key collaborators :)

**Review by aamend — APPROVED** [review]
Works for me. Please merge and let me know when I can publish next release candidate

File: src/main/scala/com/gravity/goose/spark/GooseFetcher.scala (hunk 1 above)
Comment (lamastex) [reply src/main/scala/com/gravity/goose/spark/GooseFetcher.scala:1]: Good idea. We can work towards moving goose outside as external dependency soon.

File: pom.xml (hunk 2 above)
Comment (lamastex) [reply pom.xml:17]: No worries. I will add it in the next round. Albert and Johannes have found various bugs and have finished parsers for v1 datastes.

**Pull request merged**